packages/core/src/internal/utils/string.ts

```diff
@@ -161,3 +161,12 @@ export function pluckValueFromStringUnit(val: string, unit: string) {
 export function pluckPixelValue(val: string): number {
   return !val ? 0 : pluckValueFromStringUnit(val.trim(), 'px');
 }
+
+export function interpolateNaively(template: string, dataObj: any, fallback?: string) {
```

Comment:
"naive" because it will just say `Function ()` if you try to get it to interpolate a function.